rasa/core/policies/ensemble.py

```diff
@@ -117,11 +118,13 @@ def train(
         self,
         training_trackers: List[DialogueStateTracker],
         domain: Domain,
+        interpreter: Optional[RasaCoreInterpreter],
+        output_path,
```

Comment:
since `output_path` is temporary solution, we can simply extract it form `kwargs` where it is needed, so that we don't modify all method signatures with temporary things